.github/workflows/ci.yml

```
@@ -42,6 +42,37 @@ jobs:
+    needs: tests
+    runs-on: ubuntu-latest
+    strategy: *php-matrix
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v6
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
```

actionlint (1.7.9)

property "php-version" is not defined in object type {}

(expression)